README.md

```diff
@@ -9,24 +10,29 @@ This repo is the source of <https://tidyverse.org>, and this readme tells you ho
   discuss the problem before you commit any time to it.
 
 * If you'd like to contribute a blog post, please chat with one of us first.
-  Then reading the [contributing guide](CONTRIBUTING.md)
+  Then read the [contributing guide](CONTRIBUTING.md).
 
 This repo (and resulting website) is licensed as [CC BY-SA](license.md).
 
 ## Structure
 
-The source of the website is a collection of `.md` files stored in [`content/`](content/), and turned in to html with [blogdown](https://bookdown.org/yihui/blogdown). 
+The source of the website is a collection of `.md` and `.Rmd` files stored in 
+[`content/`](content/), which are rendered for the site with 
+[blogdown](https://bookdown.org/yihui/blogdown). 
 
 * `content/*.md`: these files generate the top-level pages on the site:
   packages, learn, help, and contribute. 
   
 * `content/articles/`: these files are the tidyverse blog. New blog entries
   should be given name `year-month-slug.md`. Unfortunately this data isn't
```

Comment:
Should this be `year-month-slug.[R]?md` to stand in equally well for an `.md` and `.Rmd` post?

Reply:
✅ Yes.